installing/installing_vsphere/preparing-to-install-on-vsphere.adoc

```diff
@@ -0,0 +1,58 @@
+
+See the xref:../../architecture/architecture-installation.html#installation-process_architecture-installation[Installation process] for more information about installer-provisioned and user-provisioned installation processes.
+
+[IMPORTANT]
+====
+The steps for performing a user-provisioned infrastructure installation are provided as an example only. Installing a cluster with infrastructure you provide requires knowledge of the vSphere platform and the installation process of {product-title}. Use the user-provisioned infrastructure installation instructions as a guide; you are free to create the required resources through other methods.
+====
+
+
+=== Installer-provisioned infrastructure installation of {product-title} on vSphere
+
+Installer-provisioned infrastructure allows the installation program to pre-configure and automate the provisioning of resources required by {product-title}.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned.adoc#installing-vsphere-installer-provisioned[Installing a cluster on vSphere]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with no customization.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-customizations.adoc#installing-vsphere-installer-provisioned-customizations[Installing a cluster on vSphere with customizations]**: You can install {product-title} on vSphere by using installer-provisioned infrastructure installation with the default customization options.
+
+* **xref:../../installing/installing_vsphere/installing-vsphere-installer-provisioned-network-customizations.adoc#installing-vsphere-installer-provisioned-network-customizations[Installing a cluster on vSphere with network customizations]**: You can install {product-title} on installer-provisioned vSphere infrastructure, with network customizations. You can customize your OpenShift Container Platform network configuration during installation, so that your cluster can coexist with your existing IP address allocations and adhere to your network requirements.
+
+* **xref:../../installing/installing_vsphere/installing-restricted-networks-installer-provisioned-vsphere.adoc#installing-restricted-networks-installer-provisioned-vsphere[Installing a cluster on vSphere in a restricted network]**: You can install a cluster on VMware vSphere infrastructure in a restricted network by creating an internal mirror of the installation release content.
+ You can use this method to deploy {product-title} on an internal network that is not visible to the Internet.
+
```

Comment:
In these these bullets, you seem to refer to the IPI in different ways. Should they be consistent, or is these differences necessary?
installer-provisioned infrastructure
installer-provisioned vSphere infrastructure
VMware vSphere infrastructure

Reply:
We had a concern about everything appearing to be too similar, but I have homogenized the terminology, because it does seem like the terms were different, when really the installation method is different.